### ISSUE TITLE
Submit on editing answers and comments shows a 404 error

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#2884]
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

```
I have been writing answers and comments to queries on 
https://publiclab.org/questions/Tomas0413/06-16-2018/dark-images-can-t-get-spectrometer-to-work#a1038

When I try to edit them and click Submit after performing the edit I get 
redirected to - That page does not exist

I am using Firefox 60.0.2 (64bit) on Ubuntu 18.04

My user name is Pushpaw
```

I think this may be a Rails 5 upgrade routing issue -- @Souravirus would you mind taking a look? Thanks!!!

**Comment by Souravirus** [comment]
Yeah behind this error

**Comment by Souravirus** [comment]
Yeah editing answers is a routing error. Submitting a PR soon

**Comment by jywarren** [comment]
Solved in #2887
